### ISSUE TITLE
VSCode liveshare button

### ISSUE THREAD

**Issue by stirby** [coder/modules#263]
We should add a module that integrates with our VSCode desktop extension + liveshare. It would automatically install the VSCode liveshare extension, and add a button that opens a new session. The button would just say "Open Liveshare" and immediately provide a link to share the session.

**Comment by matifali** [comment]
I don't think there is a straightforward way to install and configure extensions in the VS Code Desktop before launching it. We may need to support this within the `coder.remote` extension after we make the connection.  
@code-asher may have more ideas here.

**Comment by code-asher** [comment]
Yeah we could add it as a dependency of our extension to make it auto-install.

**Comment by matifali** [comment]
> Yeah we could add it as a dependency of our extension to make it auto-install.

Yeah but should be configurable and only if launched through this module.

**Comment by code-asher** [comment]
That does seem tricky, I am not sure we can do optional dependencies.

Maybe we will need a second plugin that we call instead, that has Liveshare and our plugin as a dependency, and then that second plugin calls our first plugin.
